debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -486,6 +486,18 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                       .withDescription("");
 
     /**
+     * A comma-separated list of regular expressions that match source UUIDs of transactions for which we will produce
+     * DML events into Kafka. When DML events having UUIDs that do not match any of these regular expressions are
+     * processed, they will be ignored and not produced into Kafka.
+     */
+    public static final Field GTID_DML_SOURCE_INCLUDES = Field.create("gtid.dml_source.includes")
+                                                          .withDisplayName("Include GTID sources for DML production into Kafka")
+                                                          .withType(Type.LIST)
+                                                          .withWidth(Width.LONG)
+                                                          .withImportance(Importance.HIGH)
+                                                          .withDescription("We will only produce DML events into Kafka for transactions that were written on mysql servers with UUIDs included in this list.");
```

Comment:
Does it make sense to ignore DML events based upon the GTID source UUIDs defined in the `gtid.source.includes` or `gtid.source.excludes` properties?